sources/core/tests/chat_frame_info_tests.cpp

```diff
@@ -167,18 +167,22 @@ TEST(ChatFrameInfo, PostHear)
 
         phrase.from = "human1";
         phrase.text = "text1";
+        phrase.express = "express1";
```

Comment:
Here new different expressions have been added, but they are not checked.
It is needed to adjust the `TestHearer::Hear` method a little bit (located in the same file, a little bit above from here), so it would add `expression` to output and then check output below, e.g. `EXPECT_TRUE(info.GetPersonalTexts(10).contains("human1:express1:text1"));`